src/transforms/globalCssToCssModule/css/postcssToCssModulePlugin.ts

```diff
@@ -0,0 +1,135 @@
+import { AcceptedPlugin, Rule, ChildNode } from 'postcss'
+import parser, { isRoot, Selector } from 'postcss-selector-parser'
+
+const GLOBAL_TOP_LEVEL_CLASSES = ['.theme-redesign', '.theme-light', '.theme-dark']
```

Comment:
Might be useful to leave a comment here about that classes will be wrapped with `:global()`